source/reference/limitations.txt

```diff
@@ -53,6 +53,10 @@ General Limitations
 - `Queryable Execution
   <https://www.mongodb.com/docs/v6.0/core/queryable-encryption/>`__ is
   not supported.
+- If you run the :dbcommand:`renameCollection` command or the
+  :dbcommand:`drop` command on a :ref:`capped collection
+  <manual-capped-collection>`, synchronization is not supported on the
+  cluster.
```

Comment:
I think to be more precise, we can say it would be undefined behavior to do so?

Reply:
Done